dist/js/app.js

```diff
@@ -136,7 +210,9 @@ function setDownloadSection() {
       }
       else { // if there IS a matching binary for the user's OS:
         var fullOSName = OS; // defaults this variable to be the detected OS name
-        if(OS == "Win") {
+        if(OS == "Linux") {
+          fullOSName = "Linux x86-64"; // add 'x86-64'
+        } else if(OS == "Win") {
```

Comment:
You're right, this is a temporary solution for the homepage, because the homepage is still incapable of providing the other types of Linux downloads.

Could you please open an issue for this? If there are OS detection libraries out there, it would make sense to use one.